common/lib/capa/capa/templates/filesubmission.html

```diff
@@ -1,3 +1,4 @@
+<%page expression_filter="h"/>
```

Comment:
I think this template actually is used in some cases (provisional support).

http://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/exercises_tools/external_graders.html